### DISCUSSION TITLE
Simplifying getting files from sandbox as well? Default application for opening pdf?

### DISCUSSION THREAD

**Discussion by rieje** [netblue30/firejail#6004] [Q&A]
* I use Firefox and when I open a pdf, it opens it in Chromium. How can I make it open with Zathura (pdf viewer) instead? In Chromium, at least the one that Firefox launches, when I print the pdf from there, for some reason, it never prints what I see from the preview--it prints a version of the pdf with margins cut off even if I scaled it down or select "fit to printable size". I must `firejail --get` this file and then open it so I can print it properly.

* Is there a better way to `firejail --get` a file quickly from a sandbox? It involves a `firejail --list` for the id, then `firejail --ls=<id>`` to get the location of the file (even if you know where it is, often times the name of the pdf is a really long filename potentially containing all random characters), then copy that filename for `firejail --get`. AFAIK there's no way to get the full path of the file unless sometimes you can copy it from the URL if the browser is used to open the pdf (not ideal, as probably a dedicated pdf viewer is better for security and for other reasons than having the browser opening it).

**Comment by rusty-snake** [comment]
- Instead of `<id>` you can give sandboxes names at startup and use them to referre to them.
- I've wrote some scripts at https://github.com/netblue30/firejail/discussions/4746#discussioncomment-1767519 that try to make it simpler.
- You can just download into a shared folder and start your viewer from there. IDK why you want it that complicated with file transfer.
